db/migrate/20160713130940_remove_type_template_and_v_ms_filters_from_miq_search.rb

```diff
@@ -0,0 +1,43 @@
+class RemoveTypeTemplateAndVMsFiltersFromMiqSearch < ActiveRecord::Migration[5.0]
+  puts "migration"
+  puts Gem.loaded_specs.keys.inspect
+  puts defined?(GoodMigrations)
+  puts Rails.env
+  
+  class MiqSearch < ActiveRecord::Base
+    serialize :filter
+  end
+
+  puts "BEFORE MIQEXPRESSION"
+
+  TEMPLATE_FITLER_EXPR = MiqExpression.new("=" => {"field" => "VmInfra-template", "value" => "true"})
```

Comment:
Sorry for the late review @lpichler. Are you trying to get this to fail good migrations when it loads MiqExpression from the filesystem?  Because, it's in lib now so I don't think good migrations will complain about it.  Instead, try to autoload a model that we don't have a migration stub for.